src/App.tsx

```diff
@@ -1,27 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
+import { getAll, get5First, getRedGoods } from './api/goods';
+
 // import * as goodsAPI from './api/goods';
+import { Good } from './types/Good';
+
+export const App: React.FC = () => {
+  const [allGoods, setAllGoods] = useState<Good[]>([]);
+
+  const handleAllGoods = async () => {
+    const goods = await getAll();
+
+    setAllGoods(goods);
+  };
+
+  const handle5Goods = async () => {
+    const goods = await get5First();
+
+    setAllGoods(goods);
+  };
+
+  const handleColorGoods = async () => {
+    const goods = await getRedGoods();
+
+    setAllGoods(goods);
+  };
```

Comment:
For better readability and maintainability, consider renaming the function 'handleColorGoods' to 'handleRedGoods'.